### ISSUE TITLE
[JOSS Review] Import Errors

### ISSUE THREAD

**Issue by praneethd7** [EleutherAI/ccs#11]
Hey @lauritowal 

Hope all is good. Firstly, my apologies for not being timely on the review. Also, I had few overlapping suggestions with the other reviewer so I was waiting on the changes. I was trying to test the functionality and I couldn't get it running right off the bat.
![image](https://github.com/user-attachments/assets/87da4de8-019d-4612-a8cb-30035e3e4cff)

Could you help me with this? Also, it would be nice to have an example notebook to get started. You could also have it as Colab notebook. 

for reference: https://github.com/openjournals/joss-reviews/issues/6511

**Comment by lauritowal** [comment]
Hi @praneethd7 - thanks for taking the time to review!
I've added a small example notebook into the joss-paper branch: https://github.com/EleutherAI/ccs/tree/joss-paper 
The example is linked in the Readme in that branch.

Since, I've added other changes based on a previous review to that branch, it would be best if you look at that for your review instead of main. We will later merge the branch into main :)
